includes/api/order.php

```diff
@@ -119,8 +119,31 @@ function createWcOrder(WP_REST_Request $request)
     }
 
     $order = wc_get_order($orderId);
-
-    if ($order) {
+    
+    if($order){
+
+        $disableCouponFlag = false;
+
+        // Woo dynamic discount price plugin
+        if(is_plugin_active('yith-woocommerce-dynamic-pricing-and-discounts-premium/init.php')) {
+
+            foreach ($order->get_items() as $itemId => $item) {
```

Comment:
Can we get the TCD's for this PR?

Reply:
Discussed with Gaurav he said  already test case is written for this plugin so no separate TCD required , only I have to do feature testing from my side.